### PR TITLE
Crash details page: people related records table

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17317

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch locally
2. Follow [steps](https://github.com/cityofaustin/atd-vz-data/pull/1440) to start the data model collector branch and run the CRIS import ETL, or if you already have it set up use that!
3. Apply the hasura migrations and metadata from this branch
4. In the VZE go to a crash details page, check out the people table.
5. Test editing columns that can be edited. Test diff users. Check that your edits are applying to `people_edits` and not `people_cris`. Check the record you are editing in the unified `people` table. 
6. Null out an edit you made, make sure the cris value is now propagating to the top again.
7. The injury sev badges should have colors. People with fatal injuries should have editable names if your are testing as a vz-admin user. Editor/read users should not be able to see or edit victim names.
8. Editing the injury sev of a person should update the header cards and years of life lost on refresh. May address refetching the related records on edit so that the metric cards and change logs are refreshed automatically in another PR? Or could do it here

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-toolbox/data_model/python/sql_templates/people_list_view.sql
Comment (roseeichelmann) [inline atd-toolbox/data_model/python/sql_templates/people_list_view.sql:1]: from johns [people list view PR](https://github.com/cityofaustin/atd-vz-data/pull/1461) that i merged into this branch

File: atd-vze/src/queries/people.js [hunk 1]
```diff
@@ -1,145 +1,43 @@
...
-      }
-      gender {
-        gndr_id
-        gndr_desc
-      }
-      ethnicity {
-        ethnicity_id
-        ethnicity_desc
-      }
+  query FindPeople($crashId: Int!) {
```
Comment (roseeichelmann) [inline atd-vze/src/queries/people.js:4]: yay for only needing one query and one mutation now :)

File: atd-vze/src/views/Crashes/PeopleDetailsCard.js [hunk 2]
```diff
@@ -31,19 +27,17 @@ const PeopleDetailsCard = ({ isExpanded, toggleAccordion, ...props }) => {
   if (loading) return "Loading...";
   if (error) return `Error! ${error.message}`;
 
+  const primaryPersonData = data.people_list_view.filter(
+    person => person.is_primary_person === true
+  );
+
+  const personData = data.people_list_view.filter(
+    person => person.is_primary_person === false
```
Comment (roseeichelmann) [inline atd-vze/src/views/Crashes/PeopleDetailsCard.js:35]: separating out the primary person and persons data arrays by using the `is_primary_person` so they can each get their own related record. would it be more helpful to call this `otherPersonData`?

File: atd-vze/src/views/Crashes/VictimNameField.js [hunk 3]
```diff
@@ -18,30 +15,17 @@ const VictimNameField = ({
...
   ...props
 }) => {
-  const crashId = props.match.params.id;
-
-  const personId =
-    tableName === "atd_txdot_primaryperson"
-      ? row.primaryperson_id
-      : row.person_id;
-
-  const { data, refetch } = useQuery(GET_PERSON_NAMES, {
```
Comment (roseeichelmann) [inline atd-vze/src/views/Crashes/VictimNameField.js:30]: dont need to query for just the person names, we get them from the main `people_list_view` now

File: atd-vze/src/views/Crashes/peopleDataMap.js [hunk 4]
```diff
@@ -68,34 +67,32 @@ export const primaryPersonDataMap = [
...
+        lookup_desc: "label",
         format: "select",
-        lookupOptions: "atd_txdot__gndr_lkp",
-        lookupPrefix: "gndr",
+        lookupOptions: "lookups_gndr_lkp",
         updateFieldKey: "prsn_gndr_id",
         mutationVariableKey: "personId",
       },
-      ethnicity: {
+      drvr_ethncty_lkp: {
```
Comment (roseeichelmann) [inline atd-vze/src/views/Crashes/peopleDataMap.js:90]: @johnclary this is the only lookup table i could find for ethnicity. any idea why its prefixed with `drvr`? is this the proper lookup table for the `prsn_ethnicity_id` field?

File: atd-vze/src/views/Crashes/RelatedRecordsTable.js [hunk 5]
```diff
@@ -231,7 +232,6 @@ const RelatedRecordsTable = ({
                                       handleInputChange(e, updateFieldKey)
                                     }
                                   >
-                                    <option value={""}>NO DATA</option>
```
Comment (roseeichelmann) [inline atd-vze/src/views/Crashes/RelatedRecordsTable.js:234]: this was just for the suspected unhoused field, but now that this field is not nullable and defaults to false we dont need this `no data` option

File: atd-vze/src/views/Crashes/peopleDataMap.js (hunk 4 above)
Comment (johnclary) [reply atd-vze/src/views/Crashes/peopleDataMap.js:85]: yes! this is one of those that don't match the naming convention we expected. i verified this by checking the CRIS extract `xml` files back when we were researching the lookup tables.

File: atd-vze/src/views/Crashes/PeopleDetailsCard.js (hunk 2 above)
Comment (johnclary) [reply atd-vze/src/views/Crashes/PeopleDetailsCard.js:35]: or `nonPrimaryPersonData` to be super explicit. not a big deal tho 🙏

**Review by johnclary — APPROVED** [review]
this works great and has lot of nice code cleanup. awesome! 🚢

**Pull request merged**